megamek/src/megamek/common/weapons/CapitalMissileBearingsOnlyHandler.java

```diff
@@ -432,16 +431,17 @@ public void convertHexTargetToEntityTarget(Vector<Report> vPhaseReport) {
                 inArc.add(a);
             }
         }
+
         if (inArc.isEmpty()) {
-            newTarget = aaa.getTarget(game);
```

Comment:
This seems like a weird change to make. Is newTarget initialized somewhere earlier in the method?

Reply:
It's setting a method variable immediately before a return, so it's setting a variable then immediately clearing that state.